buildchain/buildchain/iso.py

```diff
@@ -126,6 +126,21 @@ def task__iso_render_bootstrap() -> types.TaskDict:
     ).task
 
 
+def task__add_upgrade() -> type.TaskDict:
```

Comment:
We now have a lot of duplication between `task__iso_add_iso_manager`, `task__add_upgrade` and `task__iso_add_node_manifest`.
We should refactor this (but you don't need to do it here, I'll open a debt issue for that)

Reply:
ok

Reply:
Tracked in #1439